app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -28,19 +28,6 @@ def textual_group_status
     )
   end
 
-  def textual_group_component_statuses
```

Comment:
is there also list somewhere of all textual groups to call?

Reply:
the only list I found is here: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/ems_container_controller.rb#L119
but it does not contain `component_statuses` I think it was already removed in the past when there was a decision to hide it (UI only). 
cc @simon3z

Reply:
it was hidden in https://github.com/ManageIQ/manageiq/pull/8609 and then probably didn't make it to the list above :)

Reply:
Oh, right!